### PR TITLE
Fix sticky position

### PR DESCRIPTION
**Pull request by t-chatoyan**

### CODE REVIEW

**Comment by ridge-kimani** [comment]
It's misplaced in mobile devices.
Screenshot 2024-01-10 at 11 52 38

**Comment by t-chatoyan** [comment]
> It's misplaced in mobile devices. Screenshot 2024-01-10 at 11 52 38

I have fixed this issue

File: assets/styles/docs.scss [hunk 1]
```diff
@@ -142,6 +142,32 @@ $bd-gutter-x: 3rem;
...
+        }
+    }
+    .bd-main {
+        .bd-toc {
+            display: none !important;
+
+            @media (max-width: 992px) {
+                display: block !important;
+                grid-area: toc !important;
+            }
```
Comment (ridge-kimani) [inline assets/styles/docs.scss:166]: Do we need `!important`? I don't see any display overrides that would prevent it from rendering as required.

**Comment by brian-mulier-p** [comment]
close in favor of https://github.com/kestra-io/docs/pull/773 to handle all margins in a better way

**Pull request closed without merging**